public/css/style.css

```diff
@@ -8,6 +8,30 @@
     src: url(/font/Poppins-Regular.ttf);
 }
 
+
+.alert {
```

Comment:
This code is not necessary, I do not see the essence of this, it was a major issue and was removed

Reply:
Bro check it now ??? I didn't change 10 files. I only git fetch upstream to update my local env before making the commit and PR. I added a screenshot of what was added in the login page